docs/resources/global_internet_bandwidth.md

```diff
@@ -0,0 +1,90 @@
+---
+subcategory: "Elastic IP (EIP)"
+---
+
+# huaweicloud_global_internet_bandwidth
+
+Manages a global internet shared bandwidth resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "enterprise_project_id" {}
+variable "bandwidth_name" {}
+variable "access_site" {}
+variable "isp" {}
+variable "bandwidth_type" {}
+
+resource "huaweicloud_global_internet_bandwidth" "test" {
+  access_site           = var.access_site
+  charge_mode           = "95peak_guar"
+  enterprise_project_id = var.enterprise_project_id
+  size                  = 300
+  isp                   = var.isp
+  name                  = var.bandwidth_name
+  type                  = var.bandwidth_type
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `access_site` - (Required, String, ForceNew) Specifies the access site name.
+  Please refer to `data.hauweicloud_global_eip_pools` to get available `access_site` in your account.
+  Changing this creates a new resource.
+
+* `charge_mode` - (Required, String) Specifies the charge mode, only support **95peak_guar** now.
```

Comment:
we can not sure if it only supports **95peak_guar**, suggest to writing like this:
 ```suggestion
* `charge_mode` - (Required, String) Specifies the charge mode, for example, **95peak_guar**.
```